scripts/links_checker/tests/common/checker/links_checker_test.dart

```diff
@@ -0,0 +1,47 @@
+import 'dart:io';
+
+import 'package:links_checker/common/checker/links_checker.dart';
+import 'package:links_checker/common/exception/links_checker_exception.dart';
+import 'package:mockito/mockito.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group("LinksChecker", () {
+    const validUrl = 'http://github.com/platform-platform/monorepo/blob/master';
+
+    // This link is concatenated because the link check will fail on this link.
+    const invalidUrl =
+        'http://github.com/platform-platform/monorepo/blob' + '/test';
```

Comment:
```suggestion
    const url = 'http://github.com/platform-platform/monorepo';
    
    const validUrl = '$url/master';
    const invalidUrl = '$url/test';
```
Also, should we improve this test data with a greater number of different URLs to widely cover regexp within a `LinksChecker` and that it parses all possible monorepo links?